src/CircuitBreaker.ts

```diff
@@ -49,6 +51,10 @@ export class CircuitBreaker extends EventEmitter {
         return this._aggregateState;
     }
 
+    get cbErr(): boolean {
```

Comment:
I am not sure to see where this is used? We indeed set it and have a getter, but is it meant to be used by the project using the lib? If yes, can we document the function, or better name it so we understand what it means? (like, "probesFailed")?

Also, the cbError is set to false in _evaluate at the beginning of the function, leaving a time window where there will be not reported error, although we did not confirm (yet) that we could check all the probes properly: can we only set it back to `false` after a successful evaluation of the probes?

Reply:
This getter is indeed used here to increment the counter on the BB side : https://github.com/scality/backbeat/pull/2516/files#diff-07825dc8c7adf67b6bbea289468e3446d7646ec1c8f5b9c361b381807fb2ce28R58-R60

For clarity sake the name has been changed here : https://github.com/scality/breakbeat/pull/11/commits/ca4687064a42c0cdca6bda864a6a94c01d0add5c#diff-359e4042f12fa2b33c080ef07d2ff300cdd9137a3f55c0dc89b5d823527da29fR54

Here for the setting of the cbErro after the evaluation is successful : https://github.com/scality/breakbeat/pull/11/commits/ca4687064a42c0cdca6bda864a6a94c01d0add5c#diff-359e4042f12fa2b33c080ef07d2ff300cdd9137a3f55c0dc89b5d823527da29fR76-R78